src/gui/editors/SongEditor.cpp

```diff
@@ -239,6 +243,9 @@ SongEditor::SongEditor( Song * song ) :
 					this, SLOT( scrolled( int ) ) );
 	connect( m_song, SIGNAL( lengthChanged( int ) ),
 			this, SLOT( updateScrollBar( int ) ) );
+	connect( m_leftRightScroll, SIGNAL(valueChanged(int)),this, SLOT(updateRubberband()));
+	connect( contentWidget()->verticalScrollBar(), SIGNAL(valueChanged(int)),this, SLOT(updateRubberband()));
```

Comment:
There's whitespace after the first opening parenthesis, but there isn't one before the last closing parenthesis. It looks a bit inconsistent to me.